tests/test_kernels/lower_memory_free/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "test_kernel_lower_memory_free"
+version = "0.1.0"
+authors = ["Philipp Oppermann <dev@phil-opp.com>"]
```

Comment:
Please either add yourself as the author or remove the `authors` key.

Reply:
yeah, you're right. I should have changed that. Done